package.json

```diff
@@ -12,7 +12,7 @@
   "dependencies": {},
   "devDependencies" : {
     "babel-eslint": "6.1.2",
-    "eslint": "^2.0.0",
+    "eslint": "^2.10.2",
```

Comment:
we can remove it from this PR, I wanted to run test in this repo against the same version we're tracking